models.py

```diff
@@ -10,7 +10,7 @@
 from playhouse.signals import Model, post_delete, pre_delete
 from uuid import uuid4
 
-from exceptions import InsufficientAvailabilityException
+from exceptions import InsufficientAvailabilityException, WrongQuantity
```

Comment:
come detto a voce questo in realtà è un unico caso, per cui si può usare la InsufficientAvailabilityException senza bisogno di creare una nuova eccezione